ignite/contrib/metrics/regression/median_relative_absolute_error.py

```diff
@@ -31,6 +31,15 @@ class MedianRelativeAbsoluteError(_BaseRegressionEpoch):
 
     __ https://arxiv.org/abs/1809.03006
 
+    Args:
```

Comment:
It misses `device` argument in `__init__`

Reply:
removed `device`